runtimes/pythonrt/py.go

```diff
@@ -0,0 +1,186 @@
+package pythonrt
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+	"io"
+	"io/fs"
+	"net"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+var (
+	//go:embed ak_runner.py
+	runnerPyCode []byte
+
+	//go:embed requirements.txt
+	requirementsData []byte
+)
+
+func createTar(fs fs.FS) ([]byte, error) {
+	var buf bytes.Buffer
+	w := tar.NewWriter(&buf)
+	if err := w.AddFS(fs); err != nil {
+		return nil, err
+	}
+
+	w.Close()
+	return buf.Bytes(), nil
+}
+
+type execInfo struct {
+	Exe     string
+	Version string
+}
+
+func pyExecInfo(ctx context.Context) (execInfo, error) {
```

Comment:
is this used only in test?

Reply:
We log it. It'll help us know what version of Python users are running. And once we have user metrics implemented ...